docs/tutorials/running-the-dal-as-a-weeklynet-baker-in-5-steps/get-octez.md

```diff
@@ -0,0 +1,6 @@
+## Step 1: get a Weeklynet-compatible Octez version
```

Comment:
Would you like to take the instructions I wrote for Weeklynet here? https://github.com/trilitech/tezos-developer-docs/pull/264/files#diff-884b105a7c433912e6bc2b905b84aef39c0d7ae0a25b28f6c852a5512a682a1cR33